ingest.py

```diff
@@ -1,316 +0,0 @@
-            initial_header.
-        """
-        header = self._initial_header + header_add
-        header[9] = 'Home'
-        header.insert(11, 'WinLoss')
-        return without(header, 'FGP', 'FTP', 'TPP')
-
-    def _table_to_db(self, season, table, header):
-        """
-        Adds all gamelogs in a table to the database.
-
-        :param season: The season type of the gamelog, either 'playoff' or
-            'regular'.
-        :param table: A basketball-reference stats table.
-        :param header: Header of the table.
-        """
-        gamelogs = []
-        for row in table.find_all('tr'):
-            stat_values_add = self._stat_values_parser(row.find_all('td'))
-            # Skip headers which are found throughout the table.
-            if not stat_values_add:
-                continue
-            stat_values = (
-                self._initial_stat_values +
-                [season] +
-                stat_values_add
-            )
-            gamelogs.append(dict(izip(header, stat_values)))
-        self._insert_gamelogs(gamelogs)
-
-    @staticmethod
-    def is_number(s):
-        """
-        Checks if a string is a number.
-
-        :param s: String to check whether is a number or not.
-        :returns: A boolean of whether or not its a number.
-        """
-        try:
-            float(s)
-            return True
-        except ValueError:
-            return False
-
-    def _stat_values_parser(self, cols):
-        """
-        Returns a list of values which change or skip the col strings based
-        on their content.
-
-        :param cols: Each column being a stat value in a row.
-        :returns: The stat values after they are parsed.
-        """
-        def get_val(i, col):
-            text = str(col.get_text())
-            conversion = self._conversions.get(i)
-            if conversion:
-                return conversion(text)
-            if self.is_number(text):
-                return float(text)
-            return text
-        return filter(notnone, (get_val(i, col) for i, col in enumerate(cols)))
-
-
-class GamelogIngester(Ingester):
-    def __init__(self, url):
-        self._url = url
-        path_components = urlparse(self._url).path.split('/')
-        self._initial_stat_values = [
-            find_player_name(path_components[3]),  # Player
-            path_components[3],  # PlayerCode
-            path_components[5]  # Year
-        ]
-        self._conversions = {
-            2: self._date_conversion,
-            5: self._home_conversion,
-            7: self._winloss_conversion,
-            12: self._percent_conversion,
-            15: self._percent_conversion,
-            18: self._percent_conversion
-        }
-        self._initial_header = ['Player', 'PlayerCode', 'Year', 'Season']
-        self._payload = None
-        self._regular_id, self._playoff_id = 'pgl_basic', 'pgl_basic_playoffs'
```

Comment:
I usually make these into constants for the class, but I don't see a downside to making them private instance variables either. Just a thought.